### ISSUE TITLE
Audio Title Not Appearing

### ISSUE THREAD

**Issue by jpcarpenter** [vidstack/player#1384]
### Current Behavior:

Audio title isn't shown in default state regardless of explicitly setting it:

``` javascript
<media-player
    v-if="src"
    class="player"
    view-type="audio"
    :title="title ?? 'Audio File'" // explicitly passing variable with fallback
    :src="src"
    crossOrigin
    playsInline
    @can-play="onCanPlay"
    ref="$player"
  >
    <media-provider></media-provider>
    <media-title></media-title>
    <media-audio-layout color-scheme="dark" />
  </media-player>
```

I've also noticed the examples provided are also not showing the titles either: https://stackblitz.com/edit/vidstack-examples-vqc1yj?file=src%2FPlayer.vue&showSidebar=1&title=Vidstack%20Player%20-%20Vue%20(Default%20Layout)

### Expected Behavior:

For audio title to be shown, such as on the older documentation site (https://old.vidstack.io/):
<img width="1109" alt="Screenshot 2024-08-08 at 1 49 28 PM" src="https://github.com/user-attachments/assets/f34d7bf1-ead3-4142-90f1-b82ad4496e75">

### Steps To Reproduce:

Reference example provided in documentation: https://stackblitz.com/edit/vidstack-examples-1kro9l?file=src%2Fmain.ts&showSidebar=1&title=Vidstack%20Player%20-%20Vue%20(Default%20Layout)

### Environment:

- Framework: Vue 3
- Meta Framework: Nuxt 3
- Node: 18.13.0
- Browser: Chromium Engine Version 127.0.6533.100

**Comment by mihar-22** [comment]
In my testing it's working as expected. The examples are outdated and need to be updated. Make sure on your end you're on the latest version and you can also prefer setting the prop rather than attribute so Nuxt doesn't complain `.title="title"`.

**Comment by jpcarpenter** [comment]
Thanks for the response @mihar-22. I've updated to the latest version and have also tried setting the prop instead of the attribute as you mentioned, but have the same issue. 

I've actually spotted what's happening. For performance reasons, I'm rendering the player conditionally based on some user interaction (turns out loading ~250 audio files/player components takes a while to load). It appears whenever I remove the conditional logic, the titles appear correctly. Would you happen to have a recommended solution for how to conditionally render the player? 

I'm rendering the component which has the player with a `v-if` like such:
``` javascript
<EpisodePlayer
  v-if="episode.initPlayer" // conditional logic happening here
  :key="episode.title"
  :src="episode?.enclosure.url"
  :title="episode.title"
/>
```

And within the `EpisodePlayer` component, I'm rendering the component like my example in the initial post. 


I can even verify the title prop from the parent component is valid within the `onMount` lifecycle hook inside the `EpisodePlayer` component, but yet the player doesn't seem to pick it up:

```javascript

interface MediaPlayerProps {
  title: string;
  src: string;
  pausePlayer: boolean;
}

const props = defineProps<MediaPlayerProps>();

onMounted(() => {
  console.log("Player mounted:", props.title); // logs the expected title, but is empty within the component
});
```

Every other aspect of the player works as expected, only the title doesn't appear. Any suggestions appreciated 🙏

**Comment by mihar-22** [comment]
Thanks for the clarification, found the bug. Pushing out fix now in `1.12.8@next` :)

**Comment by jpcarpenter** [comment]
That fixed it! Thanks for the update, and I look forward to using vidstack for future projects!
